src/generated/version.ts.tmpl

```diff
@@ -0,0 +1 @@
+export const packageVersion = 'PACKAGE_VERSION';
```

Comment:
I wonder if we can move reading `version` from `package.json` to the `compat` part and to the `constants` as well so that we won't need this separate build step, wdyt?

Reply:
This is not possible. As discussed offline, JSON cannot be imported properly in ESM without Node 17, so we need to hardcore the version with a separate build step.

Reply:
Hihiiim it is interesting, wdyt of reading it as a file and parsing it as json?

Reply:
We've talked offline (online?),  @jrandolf explained to me that the reason it doesn't work well is, let's say we have `fs.read("./package.json");` in `compat.ts` and say an app that uses Webpack bundles their app with Puppeteer dependency into `dist/dist.js`

Since the path to `./package.json` now points to `dist/package.json`, the `fs.read` won't work.